config/config.go

```diff
@@ -153,6 +158,7 @@ type Config struct {
 	AgentLogsConfig  AgentLogsConfig    `yaml:"agentLogs" json:"agentLogs"`
 	LocalModeConfig  LocalModeConfig    `yaml:"localMode" json:"localMode"`
 	InspectionConfig InspectionConfig   `yaml:"inspection" json:"inspection"`
+	JWTProvider     JWTProviderConfig  `json:"jwt_provider_config"`
```

Comment:
How do we set this from YAML config? Also it's probably fine if we make this hardcoded like the rest of the ports. It's doesn't attach to localhost anyways and sufficient if the other containers know what port to dial in the Docker network. Can we remove this and the `JWTProviderConfig` type?

Reply:
i added that one to keep it same as the rest of services, expect it to fallback to default for most cases, just to ease the testing process (prevents enforcing using the same port etc.), i don't have strong opinions on it though, editing

Reply:
I can't think of a good reason why port 8515 would be occupied. Also maybe we can change it to 8546 just to make it close to the range of the proxy container's port.